data/human/syndicate jobs.txt

```diff
@@ -529,3 +529,166 @@ mission "Return Syndicate Prisoners [2]"
 	on complete
 		payment 5000 100
 		dialog "The labor organizers are almost skeletally thin from their imprisonment. One of them raises his face to the sky as though showering in sunlight. You receive your payment of <payment>."
+
+
+mission "Syndicated Shipyards Shipment [0]"
```

Comment:
Why do the Lovelace missions get pirates, but not these? Foundry is much closer to pirate space, and Syndicate Security is, in general, weaker than the Navy.

Reply:
Ditto.